src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -30,8 +30,7 @@ public Driver create(Driver driver) {
             }
             return driver;
         } catch (SQLException throwable) {
-            throw new DataProcessingException("Couldn't create "
-                    + driver + ". ", throwable);
+            throw new DataProcessingException("Couldn't create " + driver, throwable);
```

Comment:
"Couldn't create driver " would be better

Reply:
edited it ✔️